src/angular-business/dialog/dialog-container/dialog-container.component.ts

```diff
@@ -47,56 +46,59 @@ export function throwDialogContentAlreadyAttachedError() {
   // Using OnPush for dialogs caused some G3 sync issues. Disabled until we can track them down.
   changeDetection: ChangeDetectionStrategy.Default,
   animations: [DIALOG_ANIMATIONS.slideDialog],
+  host: {
+    class: 'sbb-dialog-container',
+    tabindex: '-1',
+    'aria-modal': 'true',
+    '[attr.id]': 'id',
+    '[attr.role]': 'config.role',
```

Comment:
I thought this does not work. Did you test in a browser?

Reply:
This certainly works. however 'attr.role' probably does not work

Reply:
sorry I did remember the case i wrote 'attr.role': '...'. But yes, [attr.role] is working for sure :)